packages/lvs-types/src/v1/verificationJob.ts

```diff
@@ -0,0 +1,125 @@
+import { z } from 'zod';
+import { zParsedDate } from './zodHelpers';
+import { ISoundRecordingVerificationV1 } from './soundRecording';
+import { LicenseSourceV1 } from './license';
+
+/**
+ * The current status of the verification job.
+ */
+export enum VerificationJobStatusV1 {
+  //The initial status given to the verification job
+  Created = 'Created',
+  //The audio file is being downloaded and converted
+  Downloading = 'Downloading',
+  //The audio file currently being put through automatic song detection
+  Matching = 'Matching',
+  //The automatic matching has been completed, the license data will start downloading
+  Matched = 'Matched',
+  //All automatic processes have been completed
+  Completed = 'Completed',
+  //The end user has confirmed the detected songs are correct
+  Confirmed = 'Confirmed',
+  //Something has gone wrong, see the error message for more details
+  Errored = 'Errored'
+}
+
+export type VerificationJobLicenseDetailsV1 = z.infer<typeof verificationJobLicenseDetailsV1>
+export const verificationJobLicenseDetailsV1 = z.object({
+  //The type of license that is being uploaded.
+  //For 'ClicknClear' and 'Pending License' do not supply any license details.
+  //ClicknClear licenses are fetched automatically based on the users email address
+  licenseSource: z.nativeEnum(LicenseSourceV1),
+  //The name of the license source eg 'Other Platform'
+  licenseSourceName: z.string(),
+  //A list of URLs to download any of the relevant license files
+  licenseFileURLs: z.string().url().array()
+});
+
+export const verificationJobTagSchemaV1 = z.string().min(2).max(255);
+
+/**
+ * Used in the Create Verification Job Endpoint to create a verification job
+ */
+export type VerificationJobCreateV1 = z.infer<typeof verificationJobCreateSchemaV1>
+export const verificationJobCreateSchemaV1 = z.object({
+  //The name of the Team/Athlete/Licensee
+  name: z.string(),
+  //The email address attached to any licenses the Team/Athlete/Licensee might have
+  email: z.string().email(),
+  //The territories required by organisation, for example the territories the event is happening in: ["US", "GB"]
```

Comment:
NIT: `by organisation` -> `by the organisation`

Reply:
Resolved: a25f9a38c07b66424e9fe08fedfcc903fce503e1